sushy/redfish_interop_profiles/ServiceRoot.v1_0_1.json

```diff
@@ -0,0 +1,75 @@
+{
+    "SchemaDefinition": "OpenstackSushyRedfishInteroperabilityProfile.v1_0_0",
+    "ProfileName": "ServiceRootProfile",
+    "ProfileVersion": "1.0.1",
+    "Purpose": "Specifies the OpenStack sushy Redfish client requirements of a Redfish service implementation, typically offered by a baseboard management controller (BMC)",
+    "OwningEntity": "Dell Inc.",
+    "ContactInfo": ["richard.pioso@dell.com", "shrivastava_pranjali@dellteam.com", "digmabar_patil@dellteam.com"],
+    "Protocol": {
+        "MinVersion" : "1.0.0",
+        "Discovery" : "Recommended",
+        "HostInterface" : "Recommended"
+    },
+    "Resources": {
+        "ServiceRoot": {
+            "PropertyRequirements": {
+                "Chassis": {
+		    "ReadRequirement": "Recommended"
+		},
+                "Managers": {
+		    "ReadRequirement": "Recommended"
+		},
+                "SessionService": {
+		    "ReadRequirement": "Recommended"
+		},
+                "UUID": {
+                    "ReadRequirement": "IfImplemented"
+                },
+                "Product": {
+                    "ReadRequirement": "Recommended"
+                },
+                "RedfishVersion" : {
+                    "ReadRequirement": "Recommended"
+                },
+                "Oem" : {
+                    "ReadRequirement": "Recommended"
+                },
+                "CompositionService" : {
+                    "ReadRequirement": "IfImplemented"
```

Comment:
I have tried changing 'ReadRequirement' to 'Recommended' but validation fails in this case as the property mentioned as 'CompositionService'  is absent in the output json of /redfish/v1/. Please suggest.

Reply:
Perhaps the tool has discovered a prospective interoperability issue in the Redfish service implementation being validated \o/

Please share the validator's relevant output? Does it simply report this issue and proceed onto the rest of the validation or does it abruptly terminate when it encounters this?

Reply:
I have changed ReadRequirement to Recommended

Reply:
@spranjali, I spoke with @mraineri about the `CompositionService` property. Since sushy does not provide a way to compose systems from disaggregrated hardware, we can leave it out entirely. It could be added later, when the needed support is offered by sushy.

While the service you have been testing makes the link available, other services may not. And that would cause no issues with sushy interoperability. By leaving it out, the validator tool would not emit a false negative warning, which I believe is the behavior we want.

Reply:
Removed CompositionService property and the validator tool is working as expected where no warnings or information logs are seen w.r.t. CompositionService